conf/logback.xml

```diff
@@ -58,13 +58,13 @@
     <logger name="play.core" level="OFF"/>
     <logger name="org.asynchttpclient" level="OFF"/>
 
-    <logger name="application" level="DEBUG"/>
+    <logger name="application" level="INFO"/>
```

Comment:
Will this make us lose Debug logs in Pre-Prod environments? They have been helpful in the past.

Reply:
yep - another accidental change, will revert that.